network_setup.sh

```diff
@@ -109,7 +134,7 @@ $MIRTE_SRC_DIR/mirte-install-scripts/usb_ethernet.sh
 # be generated on first boot (so not when generating
 # the image in network_setup.sh)
 if [ ! -f /etc/ssid ] || [[ $(cat /etc/hostname) == "Mirte-XXXXXX" ]]; then
-	UNIQUE_ID=$(openssl rand -hex 3)
+	UNIQUE_ID=$(tr -cd "1-9A-F" </dev/urandom | head -c 6)
```

Comment:
This one should be removed from this PR

Reply:
Hmm, some strange things with git, now that change is gone